htdocs/comm/propal/class/propal.class.php

```diff
@@ -1680,7 +1680,7 @@ public function fetch($rowid, $ref = '', $ref_ext = '', $forceentity = 0)
 				$this->lines = array();
 
 				// Lines
-				$result = $this->fetch_lines();
+				$result = $this->fetch_lines(0, (getDolGlobalInt('MAIN_MULTILANGS') ? 1 : 0));
```

Comment:
For oerformance purpose, we should load translations only when needed. So load should be done into the code for odf generation only.

Reply:
Like that ?